partner_event/models/res_partner.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2014 Tecnativa S.L. - Pedro M. Baeza
+# Copyright 2015 Tecnativa S.L. - Javier Iniesta
+# Copyright 2016 Tecnativa S.L. - Antonio Espinosa
+# Copyright 2016 Tecnativa S.L. - Vicent Cubells
+# Copyright 2020 Tecnativa S.L. - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    event_registration_ids = fields.One2many(
+        string="Event registrations",
+        comodel_name="event.registration",
+        inverse_name="attendee_partner_id",
+    )
+    registration_count = fields.Integer(
+        string="Attendances", compute="_compute_registration_count", store=False
+    )
+
+    @api.depends("event_registration_ids", "event_registration_ids.state")
+    def _compute_registration_count(self):
+        for partner in self:
+            partner.registration_count = len(
```

Comment:
This is confusing for me. If the field is named `registration_count`, then if I register more than once, I expect the count to reflect that. Otherwise, the field name should be `event_count` instead of `registration_count`. what do you think?

![image](https://github.com/user-attachments/assets/b856df53-7b0f-4d22-9213-97f5b84e451a)

![image](https://github.com/user-attachments/assets/75271fd2-0233-4c68-9a3c-49db7c008b21)

Reply:
Yes, is missleading. I tried to figure out the cause and it simply changed from counting the registrations to the mapped events. Let's change it just making a search_count

Reply:
![Captura de pantalla de 2024-07-18 09-20-08](https://github.com/user-attachments/assets/c3f8880f-d69b-4cbf-9a50-8dc10ee2af40)

But the field shows ‘Count of events…’ in the screenshots there are 2 different events and that’s why it shows 2.

I understand that the correct thing is to display 2.

Reply:
It's missleading. You can have many attendances per event. When you click in the button you'll see a number of records that doesn't match the number printed on the button. This is an historic error, IMO

Reply:
![Captura de pantalla de 2024-07-18 09-48-14](https://github.com/user-attachments/assets/76ee3344-f531-4dc7-b742-2689a666b064)
I don’t know, those screenshots are from the contacts module, the events module correctly displays the number of attendees even if the same attendee covers several tickets, we would then have to change the ‘Count of events…’ to something else.

Reply:
I'm not sure I understand. Simply change the method to count the number of attendees and the button tooltip text to reflect that

Reply:
Done!

Reply:
could you use `search_count` instead of `len(search)`

Reply:
Done!